### PR TITLE
React to hosting changes - updated samples

### PR DESCRIPTION
**Pull request by ajaybhargavb**
https://github.com/aspnet/KestrelHttpServer/pull/496
@pranavkm @JunTaoLuo @NTaylorMullen

### CODE REVIEW

**Comment by JunTaoLuo** [comment]
Don't know why I can't click on line numbers... but remove reference to weblistener in project.json

File: samples/ActionConstraintSample.Web/hosting.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+﻿{
+    "server": "Microsoft.AspNet.Server.Kestrel",
+    "server.urls": "http://localhost:5000;https://localhost:5001"
```
Comment (Tratcher) [inline samples/ActionConstraintSample.Web/hosting.json]: Remove server.urls.

File: samples/ActionConstraintSample.Web/project.json [hunk 2]
```diff
@@ -1,7 +1,9 @@
 ﻿{
+    "compilationOptions": {
```
Comment (pranavkm) [inline samples/ActionConstraintSample.Web/project.json]: Ctrl K - D to only have 2 space indents

File: samples/ActionConstraintSample.Web/project.json [hunk 3]
```diff
@@ -1,7 +1,9 @@
...
+        "emitEntryPoint": true
+    },
     "commands": {
-        "web": "Microsoft.AspNet.Server.Kestrel",
-        "weblistener": "Microsoft.AspNet.Server.WebListener"
+        "web": "ActionConstraintSample.Web"
```
Comment (pranavkm) [inline samples/ActionConstraintSample.Web/project.json]: Remove the weblistener dependency

File: samples/CustomRouteSample.Web/project.json [hunk 4]
```diff
@@ -1,15 +1,17 @@
...
+    "compilationOptions": {
+        "emitEntryPoint": true
+    },
+    "commands": {
+        "web": "CustomRouteSample.Web"
+    },
+    "dependencies": {
+        "Microsoft.AspNet.Mvc": "6.0.0-*",
+        "Microsoft.AspNet.Server.Kestrel": "1.0.0-*",
+        "Microsoft.AspNet.Server.WebListener": "1.0.0-*"
```
Comment (JunTaoLuo) [inline samples/CustomRouteSample.Web/project.json]: ah it works now, but remove server.weblistener or server.iis when you see it

File: samples/EmbeddedViewSample.Web/project.json [hunk 5]
```diff
@@ -1,17 +1,19 @@
...
+    },
+    "commands": {
+        "web": "EmbeddedViewSample.Web"
+    },
+    "resource": "EmbeddedResources/**",
+    "dependencies": {
+        "Microsoft.AspNet.FileProviders.Embedded": "1.0.0-*",
+        "Microsoft.AspNet.Mvc": "6.0.0-*",
+        "Microsoft.AspNet.Server.Kestrel": "1.0.0-*",
+        "Microsoft.AspNet.Server.WebListener": "1.0.0-*"
```
Comment (pranavkm) [inline samples/EmbeddedViewSample.Web/project.json]: Remove

File: samples/ActionConstraintSample.Web/hosting.json (hunk 1 above)
Comment (Tratcher) [inline samples/ActionConstraintSample.Web/hosting.json]: Everywhere.

**Comment by ajaybhargavb** [comment]
:up: :date:

File: samples/InlineConstraintSample.Web/project.json [hunk 6]
```diff
@@ -1,15 +1,17 @@
...
+    "compilationOptions": {
+        "emitEntryPoint": true
+    },
+    "commands": {
+        "web": "InlineConstraintSample.Web"
+    },
+    "dependencies": {
+        "Microsoft.AspNet.Mvc": "6.0.0-*",
+        "Microsoft.AspNet.Server.Kestrel": "1.0.0-*",
+        "Microsoft.AspNet.Server.WebListener": "1.0.0-*"
```
Comment (JunTaoLuo) [inline samples/InlineConstraintSample.Web/project.json]: weblistener :-1:

Comment (ajaybhargavb) [inline samples/InlineConstraintSample.Web/project.json]: Missed. Removed.

**Comment by ajaybhargavb** [comment]
:up: :date:

**Comment by pranavkm** [comment]
:shipit:

**Pull request merged**